cmd/verify/main.go

```diff
@@ -0,0 +1,102 @@
+/*
+Copyright 2022 The Tekton Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
```

Comment:
Instead of making 2 separate binaries, start prepping this to be a tkn plugin by making this 1 single binary with subcommands.

Reply:
Is it ok to keep these 2 binaries before tkn support? Then we depreciate/remove them. 
The tkn side may take longer time, including TEP and implementation.  Just worried we have no way to sign files

Reply:
You don't need a TEP for a tkn plugin - you just have to install the binary with the name `tkn-<plugin name>` https://github.com/chmouel/tektoncd-cli/blob/68f6ba9f026247a89b56adbb5e17e5bbade71506/cmd/tkn/main.go#L44-L61

You could still do this with 2 binaries if you wanted, though it becomes a little more annoying to distribute.

Reply:
I will add these into tkn as subcommand. Created an issue here https://github.com/tektoncd/cli/issues/1733.
So for this commit in the doc I will point to experimental for signing, and when tkn supports we can update the doc